web/src/modules/topic/components/Indicator/ClaimIndicator.tsx

```diff
@@ -16,13 +17,17 @@ export const ClaimIndicator = ({ arguableId, arguableType }: Props) => {
   const Icon = explicitClaimCount > 0 ? Article : ArticleOutlined;
 
   return (
-    <Indicator
-      Icon={Icon}
-      onClick={(event) => {
+
+    <Tooltip title="View claims">
+
+    <Indicator  Icon={Icon} onClick={(event) => {
```

Comment:
I'm noticing the spacing is off - I think prettier needs to be run `npx prettier --write .`. I can handle this if need be.

I realize the contributing guide doesn't mention prettier, nor is it set up automatically for contributors - I think I'll add a precommit hook and/or github action to have this run